frontend/src/pages/team/changeType.vue

```diff
@@ -155,6 +158,13 @@ export default {
                    !this.user.admin &&
                    this.input.teamType && this.input.teamType.properties?.billing?.requireContact
         },
+        isCurrentUnavailable () {
+            if (!this.teamTypes) {
```

Comment:
```suggestion
            if (this.teamTypes.length === 0) {
```

Reply:
`this.teamTypes` is an array, not sure if `!this.teamTypes` will work as expected

Reply:
I was handling the case where `this.teamTypes` hadn't been initialised yet - but double checking, I see it gets initialised as an empty array, so yes, this is appropriate.